server/pulp/server/async/scheduler.py

```diff
@@ -317,15 +320,60 @@ def tick(self):
         :return:    number of seconds before the next tick should run
         :rtype:     float
         """
-        ret = super(Scheduler, self).tick()
         self._failure_watcher.trim()
-
         # this is not an event that gets sent anywhere. We process it
         # immediately.
         scheduler_event = {'timestamp': time.time(),
                            'type': 'scheduler-event',
-                           'hostname': ("%s@%s" % (SCHEDULER_WORKER_NAME, platform.node()))}
+                           'hostname': ("%s@%s@%s" % (SCHEDULER_WORKER_NAME,
+                                                      str(celery_name), platform.node()))}
         worker_watcher.handle_worker_heartbeat(scheduler_event)
+
+        celerybeat_name = SCHEDULER_WORKER_NAME + "@" + str(celery_name) + "@" + platform.node()
```

Comment:
This is being calculated twice. once here in line 332 and once in lines 328-239. Compute it once above and use it in both places.